common/static/common/js/spec_helpers/page_helpers.js

```diff
@@ -1,18 +1,38 @@
-define([ // jshint ignore:line
-],
-function() {
-    'use strict';
-    var getLocationHash;
-    /**
-     * Helper method that returns url hash.
-     * @return {String} Returns anchor part of current url.
-     */
-    getLocationHash = function() {
-        return window.location.hash;
-    };
+define(["backbone"],
+    function(Backbone) {
+        'use strict';
+        var getLocationHash, installMockBackboneHistory;
 
-    return {
-        'getLocationHash': getLocationHash
-    };
+        /**
+         * Helper method that returns url hash.
+         * @return {String} Returns anchor part of current url.
+         */
+        getLocationHash = function() {
+            return window.location.hash;
+        };
 
-});
+        /**
+         * Mocks the Backbone history so that navigation doesn't change
+         * the browser's own history.
+         */
+        installMockBackboneHistory = function() {
```

Comment:
I've decided to break out this router changing into its own PR since it is unrelated to this change: https://github.com/edx/edx-platform/pull/9885